lib/screens/questionaire_screen/widgets/cover.dart

```diff
@@ -82,14 +91,14 @@ class Cover extends StatelessWidget {
               ),
             ),
             SizedBox(
-              height: height / 3.5,
+              height: sizedBoxHeight,
             ),
-            if (backFunction == null)
+            if (backFunction == null && nextFunction != null)
               PicosInkWellButton(
                 text: start,
                 onTap: nextFunction ?? () {},
               )
-            else
+            else if (backFunction != null || nextFunction != null)
```

Comment:
You are doing the same checks twice. Maybe you could consolidate, so you don't have to repeat your conditions all over the place.